tests/test_mirror.py

```diff
@@ -85,17 +85,16 @@ def remove_mirror_session(self, name):
 
     def get_mirror_session_status(self, name):
         status = ""
-        # TODO: the status of mirror session will be moved to state database
-        tbl = swsscommon.Table(self.pdb, "MIRROR_SESSION")
+        tbl = swsscommon.Table(self.sdb, "MIRROR_SESSION")
         (status, fvs) = tbl.get(name)
         assert status == True
-        assert len(fvs) == 1
+        assert len(fvs) > 0
         for fv in fvs:
             if fv[0] == "status":
                 status = fv[1]
-            else:
-                assert False
-        return status
+                return status
+        else:
```

Comment:
why this `else` is kept?

Reply:
This is part of for-else block. [http://book.pythontips.com/en/latest/for_-_else.html](http://book.pythontips.com/en/latest/for_-_else.html)
If there is no status field, then it is a logic error.

---
In reply to: [215804273](https://github.com/Azure/sonic-swss/pull/609#discussion_r215804273) [](ancestors = 215804273)

Reply:
The for/else isn't really necessary, since there is no way to break out of the loop. If the loop succeeds, the function will return, thus simply having the `assert False` line after the loop body would produce the same behavior.

Reply:
You're right. Fixed.

---
In reply to: [215820363](https://github.com/Azure/sonic-swss/pull/609#discussion_r215820363) [](ancestors = 215820363)